Demo/HYPForms/HYPSampleCollectionViewController.m

```diff
@@ -233,15 +230,7 @@ - (void)readOnly:(UISwitch *)sender
 {
     [self.dataSource disable:sender.isOn];
 
-    HYPFormTarget *target = [HYPFormTarget new];
-    target.targetID = @"image";
-    target.typeString = @"field";
-
-    if (sender.isOn) {
-        target.actionTypeString = @"hide";
-    } else {
-        target.actionTypeString = @"show";
-    }
+    HYPFormTarget *target = [HYPFormTarget hideFieldTargetWithID:@"image"];
```

Comment:
Are you sure it's always "hide"? In the earlier version it depended on sender.isOn, now it seems to be always hide.